File: C#/백준/Bronze/1871. 좋은 자동차 번호판/좋은 자동차 번호판.cs
using System;

namespace q{
    class w{
        static void Main(){
            int a = int.Parse(Console.ReadLine());
            for (int i=0;i<a;i++){
                string[] aa = Console.ReadLine().Split("-");
                string s = aa[0];
                int n1 = 0;
                int n2 = int.Parse(aa[1]);
                int p = 26*26;
                for (int j=0;j<3;j++){
                    n1+=(s[j]-'A')*p;
                    p/=26;
                }
                if(Math.Abs(n1-n2)<=100){
                    Console.WriteLine("nice");
                }
                else{
                    Console.WriteLine("not nice");
                }
            }
        }
    }
}

File: Java/백준/Bronze/1871. 좋은 자동차 번호판/좋은 자동차 번호판.java
import java.util.Scanner;
import java.lang.Math;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int q = in.nextInt();
        for (int i=0;i<q;i++){
            String[] aa = in.next().split("-");
            int a = Integer.parseInt(aa[1]);
            int p = 26*26;
            int b = 0;
            for (int j=0;j<3;j++){
                b+=(aa[0].charAt(j)-'A')*p;    
                p/=26;
            }
            if(Math.abs(a-b)<=100){
                System.out.println("nice");
            }
            else{
                System.out.println("not nice");
            }
        }
    }
}